projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -19,8 +19,13 @@
   <a routerLink="/records/documents/new" [queryParams]="{source: source, pid: pid}" class="btn btn-sm btn-primary"
     translate>Import</a>
 </div>
+<!-- duplicate button -->
+<div *ngIf="record && record.metadata && record.metadata.pid" class="float-right mt-2 mb-4 btn-duplicate">
+  <a routerLink="/records/documents/duplicate" [queryParams]="{type: 'documents', pid: record.metadata.pid}" class="btn btn-sm btn-primary"
```

Comment:
I think the right syntax is: `[routerLink]="['/records', 'documents', 'duplicate']">`
See line 108.
And it's the occasion to correct line 19 too.

Reply:
done